packages/infra/lib/surescripts-stack.ts

```diff
@@ -0,0 +1,516 @@
+}
+
+const settings: SurescriptsSettings = {
+  connectSftp: {
+    name: "SurescriptsConnectSftp",
+    entry: "surescripts-sftp-connect",
+    memory: 512,
+    timeout: connectSftpTimeout,
+  },
+  synchronizeSftp: {
+    name: "SurescriptsSynchronizeSftp",
+    entry: "surescripts-sftp-synchronize",
+    lambda: {
+      memory: 2048,
+      timeout: synchronizeSftpTimeout,
+    },
+    queue: {
+      alarmMaxAgeOfOldestMessage: Duration.hours(1),
+      maxMessageCountAlarmThreshold: 1000,
+      maxReceiveCount: 3,
+      visibilityTimeout: Duration.seconds(synchronizeSftpTimeout.toSeconds() * 2 + 1),
+      createRetryLambda: false,
+    },
+    eventSource: {
+      batchSize: 1,
+      reportBatchItemFailures: true,
+      maxConcurrency: 4,
+    },
+    waitTime: Duration.seconds(1), // Max 60 syncs/minute
+  },
+
+  sendPatientRequest: {
+    name: "SurescriptsSendPatientRequest",
+    entry: "surescripts-send-patient-request",
+    lambda: {
+      memory: 1024,
+      timeout: sendPatientRequestLambdaTimeout,
+    },
+    queue: {
+      alarmMaxAgeOfOldestMessage: Duration.hours(3),
+      maxMessageCountAlarmThreshold: 15_000,
+      maxReceiveCount: 3,
+      visibilityTimeout: Duration.seconds(sendPatientRequestLambdaTimeout.toSeconds() * 2 + 1),
+      createRetryLambda: false,
+    },
+    eventSource: {
+      batchSize: 1,
+      reportBatchItemFailures: true,
+      maxConcurrency: 4,
+    },
+    waitTime: Duration.seconds(0), // No limit
+  },
+  receiveVerificationResponse: {
+    name: "SurescriptsReceiveVerificationResponse",
+    entry: "surescripts-receive-verification-response",
+    lambda: {
+      memory: 1024,
+      timeout: receiveVerificationLambdaTimeout,
+    },
+    queue: {
+      alarmMaxAgeOfOldestMessage: Duration.hours(1),
+      maxMessageCountAlarmThreshold: 15_000,
+      maxReceiveCount: 3,
+      visibilityTimeout: Duration.seconds(receiveVerificationLambdaTimeout.toSeconds() * 2 + 1),
+      createRetryLambda: false,
+    },
+    eventSource: {
+      batchSize: 1,
+      reportBatchItemFailures: true,
+      maxConcurrency: 4,
+    },
+    waitTime: Duration.seconds(0), // No limit
+  },
+  receiveFlatFileResponse: {
+    name: "SurescriptsReceiveFlatFileResponse",
+    entry: "surescripts-receive-flat-file-response",
+    lambda: {
+      memory: 2048,
+      timeout: receiveFlatFileResponseLambdaTimeout,
+    },
+    queue: {
+      alarmMaxAgeOfOldestMessage: Duration.hours(1),
+      maxMessageCountAlarmThreshold: 15_000,
+      maxReceiveCount: 3,
+      visibilityTimeout: Duration.seconds(receiveFlatFileResponseLambdaTimeout.toSeconds() * 2 + 1),
+      createRetryLambda: false,
+    },
+    eventSource: {
+      batchSize: 1,
+      reportBatchItemFailures: true,
+      maxConcurrency: 4,
+    },
+    waitTime: Duration.seconds(0), // No limit
+  },
+};
+
+function surescriptsEnvironmentVariables(
+  surescripts: EnvConfig["surescripts"],
+  replica = true,
+  bundle = false
+): {
```

Comment:
Especially when using params like this that have types and we might be passing in true or false, prefer to use objects for args, like so.
```
function surescriptsEnvironmentVariables({
  surescripts,
  replica = true,
  bundle = false
}): {
...


// invocation
surescriptsEnvironmentVariables({ surescripts: something, replica: false })
```